README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img width="65%" src="docs/source/_static/img/evalai_logo.png"/></p>
+<p align="center" style="padding-left: 125px;"><img width="95%" src="docs/source/_static/img/evalai_logo.png"/></p>
```

Comment:
Seems like left-padding wont solve the issue since width is not same for all screen sizes. Might want to try `align` property.

Reply:
Sure.